shop/src/components/ConfirmationModal.js

```diff
@@ -95,7 +95,6 @@ const AdminConfirmationModal = ({
             if (onClose) {
               onClose()
             }
-            setState({}, true)
```

Comment:
Hi @baaae,

I reviewing this PR, I learned that this line functions as an `else` statement for the function definition between lines 91 and 98; I recommend keeping it because all components that use the `AdminConfirmationModal` component need to be updated with an `onClose` prop of their own otherwise. Examples include things like the [Delete button](https://github.com/OriginProtocol/dshop/blob/master/shop/src/pages/admin/discounts/_Delete.js) under **Discounts** and the [Publish Changes button](https://github.com/OriginProtocol/dshop/blob/master/shop/src/pages/admin/settings/deployments/_DeployButton.js).

The effect of not updating such components may be seen below:
![AdminConfirmationModal](https://user-images.githubusercontent.com/10854442/116865980-428ae400-ac28-11eb-8b80-f2ec4c1d70aa.gif)

Reply:
@phyninja Good catch! I'll add it back in for now and work out a different solution in the future.